sample_application/make_db.py

```diff
@@ -0,0 +1,4 @@
+from models import db
+from app import create_app
+with create_app().app_context():
+    db.create_all()
```

Comment:
As per what @vsudilov suggested, I would keep what you had. So, I'd keep the manage.py you had, and then add a command that can do the above. For example the manage.py could look like this:
```
from flask.ext.script import Manager, Command
from flask.ext.migrate import Migrate, MigrateCommand
from models import db
from app import create_app


class CreateDatabase(Command):
    """
    Creates the database based on models.py
    """

    def run(self):
        with create_app().app_context():
            db.create_all()

app_ = create_app()
migrate = Migrate(app_, db)
manager = Manager(app_)

manager.add_command('db', MigrateCommand)
manager.add_command('createdb', CreateDatabase())

if __name__ == '__main__':
    manager.run()
```

Now you can create the db using `manage.py createdb` and do the migrate commands with `manage.py db init`, etc., as you had implemented at the start.